src/data/category-data.jsx

```diff
@@ -0,0 +1,23 @@
+import bag from "../assets/images/bag.jpg"
+import rackets from "../assets/images/rackets.jpg"
+import shoes from "../assets/images/shoes.jpg"
+import shuttles from "../assets/images/shuttles.jpg"
+
+export const categories = [
+    {
+        category : "Rackets",
```

Comment:
suggestion: would be good if you name the  key as "name" instead of category

Reply:
Here I have given key as a ```category``` because each category will have products and products will have their respective names. So just for the sake that these two doesn't conflict I have gone with key as ```category```